### PR TITLE
SvelteKit

### PR DESCRIPTION
**Pull request by ajcwebdev**

### CODE REVIEW

**Review by Cerchie — APPROVED** [review]
I learned a lot about SvelteKit from the README! Thanks Anthony! 😸

File: with-sveltekit/src/routes/articles.json.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export async function post() {
+  const response = await fetch(STEPZEN_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `apikey ${STEPZEN_API_KEY}`,
+    },
+    body: JSON.stringify({
+      query: `{
+        getArticles(username: "ajcwebdev", per_page: 100) {
```
Comment (royderks) [inline with-sveltekit/src/routes/articles.json.js:14]: Can we make this a variable in the config file? We ask people for their Dev.to API key so it makes sense to also ask for their username. Fine to mention your username as a "default"

File: with-sveltekit/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Include your StepZen endpoint and API keys in `.env`, by copying then file `.env.example` to `.env`:
+
+```bash
+mv .env.example .env
+```
+
+And add the endpoint and API Key for Stepzen:
+
+```
+STEPZEN_ENDPOINT=
```
Comment (royderks) [inline with-sveltekit/README.md:50]: Can we show an error if someone tries to run the code without the environment variables? I ran the sample without them and had no clue what was going wrong - other than retracing my steps. A simple: "Please add your StepZen credentials to the file `.env`" will do

**Comment by royderks** [comment]
Looks great @ajcwebdev, two small requests in the review comments above. Also, I've updated your readme according to the `TEMPLATE.md` :)

Don't forget to save the previous contents of readme.md from the commit history somewhere before deleting the branch.

**Comment by royderks** [comment]
> Looks great @ajcwebdev, two small requests in the review comments above. Also, I've updated your readme according to the `TEMPLATE.md` :)
> 
> Don't forget to save the previous contents of readme.md from the commit history somewhere before deleting the branch.

And also, can we use the netlify deploy script to put a button in the readme?

**Pull request merged**